cmd/cluster-init/update_sanitize_prow_jobs.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/dispatcher"
+)
+
+func updateSanitizeProwJobs(o options) error {
+	logrus.Println("Updating sanitize-prow-jobs config")
+	filename := filepath.Join(o.releaseRepo, "core-services", "sanitize-prow-jobs", "_config.yaml")
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+	c := &dispatcher.Config{}
+	if err = yaml.Unmarshal(data, c); err != nil {
+		return err
+	}
+	updateConfig(c, o.clusterName)
+	y, err := yaml.Marshal(c)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filename, y, 0644)
+}
+
+func updateConfig(c *dispatcher.Config, clusterName string) {
+	appGroup := c.Groups[api.ClusterAPPCI]
+	jobs := appGroup.Jobs
+	jobs = append(jobs, fmt.Sprintf("pull-ci-openshift-release-master-%s-dry", clusterName))
+	jobs = append(jobs, fmt.Sprintf("branch-ci-openshift-release-master-%s-apply", clusterName))
+	jobs = append(jobs, fmt.Sprintf("periodic-openshift-release-master-%s-apply", clusterName))
+	c.Groups[api.ClusterAPPCI] = dispatcher.Group{
```

Comment:
I believe you should be able to do

```go
c.Groups[api.ClusterAPPCI].Jobs = append(c.Groups[api.ClusterAPPCI].Jobs, ....)
```

Reply:
That was my initial thought as well, but go doesn't allow that. It actually throws a compiler error. Apparently you can't append to a slice that is contained in a map. I had read an SO article on this at some point, but I lost it now. LMK if you want me to look for it.

Reply:
Let's do this, then, so the you do not have to manage each individual field of the struct in the copy:

```go
appGroup := c.Groups[api.ClusterAPPCI]
appGroup.Jobs = append(appGroup.Jobs, ...)
c.Groups[api.ClusterAPPCI] = appGroup
```

Reply:
That does work, and is a bit cleaner, thanks